benchmarks/Smalltalk/build-image-squeak.st

```diff
@@ -0,0 +1,65 @@
+| parentDir runner loader starting |
```

Comment:
Could you please provide an explanatory comment on the PR about why this is needed, what you do here, and what changes are there that go beyond avoiding incompatibilities between Squeak/Pharo?